piccolo/apps/schema/commands/generate.py

```diff
@@ -131,6 +131,41 @@ def get_foreign_key_constraint_name(self, column_name) -> ConstraintTable:
         raise ValueError("No matching constraint found")
 
 
+@dataclasses.dataclass
+class Trigger:
+    constraint_name: str
+    constraint_type: str
+    table_name: str
+    column_name: str
+    on_update: str
+    on_delete: Literal["NO ACTION", "RESTRICT", "CASCADE", "SET NULL", "SET_DEFAULT"]
+    references_table: str
+    references_column: str
+
+@dataclasses.dataclass
+class TableTriggers:
+    """
+    All of the triggers for a certain table in the database.
+    """
+
+    tablename: str
+    triggers: t.List[Trigger]
+
+    def get_column_triggers(self, column_name) -> t.List[Trigger]:
+        triggers = []
+        for i in self.triggers:
+            if i.column_name == column_name:
+                triggers.append(i)
+        return triggers
+    
+    def get_column_ref_trigger(self, column_name, references_table) -> Trigger:
```

Comment:
Nitpick - type annotation for `column_name` and `references_table`.

Reply:
roger that